sdk/storage/storage-file-datalake/test/perfstress/track-2/storageTest.spec.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressTest } from "@azure/test-utils-perfstress";
+
+import {
+  DataLakeServiceClient,
+  StorageSharedKeyCredential,
+  DataLakeFileSystemClient,
+  DataLakeDirectoryClient
+} from "../../../src";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export abstract class StorageDFSTest<TOptions> extends PerfStressTest<TOptions> {
+  datalakeServiceClient: DataLakeServiceClient;
+  fileSystemClient: DataLakeFileSystemClient;
+  directoryClient: DataLakeDirectoryClient;
+  static fileSystemName = `newfs${new Date().getTime()}`;
+  static directoryName = `newdirectory${new Date().getTime()}`;
+
+  constructor() {
+    super();
+    const connectionString = StorageDFSTest.getEnvVar("STORAGE_CONNECTION_STRING");
+    const accountName = getValueInConnString(connectionString, "AccountName");
+    const accountKey = getValueInConnString(connectionString, "AccountKey");
+    const sharedKeyCredential = new StorageSharedKeyCredential(accountName, accountKey);
+
+    this.datalakeServiceClient = new DataLakeServiceClient(
+      `https://${accountName}.dfs.core.windows.net`,
+      sharedKeyCredential
+    );
+
+    this.fileSystemClient = this.datalakeServiceClient.getFileSystemClient(
+      StorageDFSTest.fileSystemName
+    );
+
+    this.directoryClient = this.fileSystemClient.getDirectoryClient(StorageDFSTest.directoryName);
+  }
+
+  public async globalSetup() {
+    await this.fileSystemClient.create();
+    await this.directoryClient.create();
+  }
+
+  public async globalCleanup() {
+    await this.fileSystemClient.delete();
+  }
+
+  static getEnvVar(name: string) {
```

Comment:
Move this into the PerfStressTest base class.

Reply:
Done